README.md

```diff
@@ -25,11 +25,11 @@ The best demo to start with is [cp-demo](https://github.com/confluentinc/cp-demo
 
 | Demo                                       | Local | Docker | Description
 | ------------------------------------------ | ----- | ------ | ---------------------------------------------------------------------------
-| [ACLs in Cloud](security/acls) |  Y  |  N  | ACLs in Confluent Cloud <br><img src="https://cdn.confluent.io/wp-content/uploads/Secret_Protection_Feature.jpg" width="400">
+| [Beginner demo](security/acls) |  Y  |  N  | Fully scripted demo that shows how to interact with your Confluent Cloud cluster and set ACLs using the CLI <br><img src="clients/cloud/images/confluent-cloud.jpeg" width="400">
```

Comment:
How about "Cloud Intro" or "Beginner Cloud"?

Reply:
Love it.  Done.